examples/zipkin-tracing/verify.sh

```diff
@@ -13,9 +13,13 @@ responds_with \
     "Hello from behind Envoy (service 1)!" \
     "http://localhost:${PORT_PROXY}/trace/1"
 
-run_log "Test dashboard"
 # this could do with using the healthcheck and waiting
-sleep 20
+wait_for 30 bash -c "\
+         responds_with \
+         '<!doctype html>' \
+         http://localhost:${PORT_ADMIN}/zipkin/"
```

Comment:
Might be missing context, why are we  querying it twice with same query? Here and a few other places

Reply:
the most reliable way of checking that the test is ready to be run is just waiting for the test to pass

ditto elsewhere

despite trying i didnt find a way to make this wait for docker correctly - i guess i could just move these to healthchecks on the compositions

my pref is to land and iterate because these make ci less flakey and faster

Reply:
iteration sgtm.